thanos-operator.yaml

```diff
@@ -0,0 +1,50 @@
+package:
+  name: thanos-operator
+  version: 0.3.7
+  epoch: 0
+  description: Kubernetes operator for deploying Thanos
+  copyright:
+    - license: Apache-2.0
+
+environment:
+  contents:
+    packages:
+      - busybox
+      - ca-certificates-bundle
+      - go
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/banzaicloud/thanos-operator
+      tag: ${{package.version}}
+      expected-commit: 7b87b4f89dee6454ae46e9579b9a98d6d7a5e550
+
+  - runs: |
+      # Address  CVE-2022-21698  GHSA-69ch-w2m2-3vjp  GHSA-69cg-p879-7622
+      go get github.com/prometheus/client_golang@v1.11.1
+      go get golang.org/x/text@v0.3.8
+      go get golang.org/x/net@v0.7.0
+
+      go mod tidy
+
+      go build -o bin/manager main.go
+      mkdir -p ${{targets.destdir}}/usr/bin
+      install -Dm755 ./bin/manager ${{targets.destdir}}/usr/bin/manager
+
+  - uses: strip
+
+subpackages:
+  - name: "thanos-operator-compat"
+    description: "Compatibility package to place binaries in the location expected by upstream helm charts"
+    pipeline:
+      - runs: |
+          # The helm chart expects the thanos-operator binaries to be in / instead of /usr/bin
+          mkdir -p "${{targets.subpkgdir}}"
+          ln -sf /usr/bin/manager ${{targets.subpkgdir}}/manager
+      - uses: strip
+
+update:
```

Comment:
looks like the auto update CI check us failing as there are SDK releases on this repo.  You can ignore these and have the CI check go green using https://github.com/chainguard-dev/melange/blob/main/docs/UPDATE.md#ignore-versions

Reply:
what should be the value of the `ignore-regex-patterns`? I couldn't understand what it should be :(